### DISCUSSION TITLE
Better 'Kiai Time' ✨🥁✨

### DISCUSSION THREAD

**Discussion by Bryceed** [ppy/osu#23237] [Ideas]
Today it's hard to feel where the music are during the Kiai mode or not, with more discrete effects (maybe thinking in performance?), but at least the possibility to enable a more beating/vibrant Kiai mode, to the players really feels when the music enters in a more "hyped moment" of the music or not (by the interpretation of the mapper).

A suggestion is to have the lateral beating light on the menu into the gameplay - and it's nothing that needed to be 'create from scratch', just can be optionally improved for gameplay purposes.

**Comment by TheFearlessDeath500** [comment]
Personally I would feel like Kiai Time is already VERY flashy as is. I’ve seen a few Taiko no Tatsujin players try osu!taiko and they mostly complained about Kiai in particular being too flashy.
This definitely shows in stable builds too— if there is low BPM and SV, and Kiai Time is applied to a very dense patterns, it will mostly be a huge flashy mess.

**Reply by Bryceed** [reply]
I understand your point but disagree with your comment (reinforcing that I'm not talking exclusively about Taiko mode, but rather, for Kiai Time's moment in game modes, including osu!standard)

Kiai Time is an important part of the song, that increases the excitement and difficulty of the game. It's a visual effect that highlights the song's most intense moments - as least expected to be, by the mappers. Taiko no Tatsujin players may not be used to Kiai Time, but that doesn't mean it's bad or unnecessary.

So, thinking about it, in the individual preferences of each player, I suggested it as an enabling option. Besides, the suggestion doesn't escape beyond what happens today in osu!stable, during Kiai Time.

Let the players choose not to use is much better than not being able to use it because, basically, you don't have any option to do it.

**Reply by TheFearlessDeath500** [reply]
I see, I'd agree with this actually- making Kiai note flashes should be an option to toggle as well.

**Comment by DarkGradient** [comment]
Yea, this option shall be present. 
Ability to toggle kiai time flashing in gameplay
And ability to control "brightness" of kiai time in gameplay

**Comment by Bryceed** [comment]
Totally agree @DarkGradient. 

Currently, 1 year later, when Kiai Time starts, there's a particle effect resembling fireworks, which is more intrusive and... it's there - with option to disable. **This effect doesn't represent the event's full duration and doesn't indicate when Kiai's ends.**

While Kiai Time aims to highlight significant moments in a beatmap, generally chorus or hard beats moments, the current effects can be either too subtle or overly intrusive. 

Customizing these effects could offer a more balanced and player-preferred gaming experience, especially for those sensitive to intense visuals.
